index.js

```diff
@@ -309,6 +333,7 @@ class ServerlessAppsyncPlugin {
           },
           Policies: [
             {
+              // TODO: Why are inline policies getting fine-grained policy names? There is no benefit to this.
               PolicyName: this.getDataSourceCfnName(ds.name) + "Policy",
```

Comment:
@sid88in @cbaron Is there any reason to the above I'm missing? Inline policies that aren't a Policy resource don't really have a requirement to have datsource specific names, being attached to only one parent role.

Reply:
@ZY-Ang, I don't have much context here.  Its a good question.  My guess is that @sid88in wanted to ensure descriptive policy names.

Reply:
@ZY-Ang @cbaron I wrote that line. 
I am not sure I understood your question though...
`PolicyName` is a required field in cloudformation
https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-iam-policy.html#cfn-iam-policy-policyname
I just gave it a name related to the Datasource it refers to as a reference.

Or did I miss something?

Reply:
@bboure I was just confused. It's like arguing `pineappleRole.penPolicy` vs `pineappleRole.penPineapplePolicy`. Although the latter would make sense if it was a policy resource and not an inline policy

Reply:
@ZY-Ang I am ok with changing it. I used this for no particular reason other than it was required.  What do you suggest?  Something like `defaultPolicy` or `root` (Like shown [here](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-iam-role.html)) or something?